includes/sanitizers/class-amp-meta-sanitizer.php

```diff
@@ -137,10 +137,9 @@ protected function ensure_viewport_is_present() {
 		 */
 		$viewport_tag      = $this->meta_tags[ self::TAG_VIEWPORT ][0];
 		$viewport_content  = $viewport_tag->getAttribute( 'content' );
-		$viewport_settings = array_map( 'trim', explode( ',', $viewport_content ) );
+		$viewport_settings = array_filter( array_map( 'trim', explode( ',', $viewport_content ) ) );
 		$width_found       = false;
 
-		// @todo The invalid/missing properties should raise validation errors. See <https://github.com/ampproject/amp-wp/pull/3758/files#r348074703>.
 		foreach ( $viewport_settings as $index => $viewport_setting ) {
 			list( $property, $value ) = array_map( 'trim', explode( '=', $viewport_setting ) );
```

Comment:
A missing change is to put the trimmed `$property` and `$value` into the `$viewport_settings`, correct? This would then address https://github.com/ampproject/amp-wp/pull/4137#issuecomment-578467681

Reply:
I made the change to `AMP_Tag_And_Attribute_Sanitizer` in https://github.com/ampproject/amp-wp/pull/4137/commits/c4a5581701e21d9ed02d40ff25533f3e8d728a0c before seeing this. It definitely would be suited for `AMP_Meta_Sanitizer`.

Reply:
Done via https://github.com/ampproject/amp-wp/pull/4137/commits/b25781997dac38d6337bfd89a4752f285981de2d.